crates/evm/src/utils.rs

```diff
@@ -39,14 +39,14 @@ pub fn h256_to_u256_le(storage: H256) -> U256 {
 
 /// Small helper function to convert revm's [B160] into ethers's [H160].
 #[inline]
-pub fn b160_to_h160(b: revm::primitives::B160) -> ethers::types::H160 {
-    ethers::types::H160(b.0)
+pub fn b160_to_h160(b: alloy_primitives::Address) -> ethers::types::H160 {
+    H160::from_slice(b.as_slice())
```

Comment:
This wouldve been much easier with a conversion trait tbh

```rust
trait Conv {
    type To;
    fn conv(self) -> Self::To;
}

impl Conv for ethers::types::H160 {
    type To = alloy_primitives::B160;
    ...
}

impl Conv for alloy_primitives::B160 {
    type To = ethers::types::H160;
    ...
}
```

Reply:
yeah for sure, these were just introduced a while ago and decided to reuse them—I think we could switch to the conversion trait later if we need it (hopefully not for long)